instaclustr/structs.go

```diff
@@ -217,6 +224,20 @@ type GCPVPCPeering struct {
 	StatusCode  string        `json:"statusCode"`
 }
 
+type AzureVPCPeering struct {
+	ID                 string        `json:"id"`
+	PeerSubscriptionId string        `json:"peerSubscriptionId"`
+	ClusterDataCentre  string        `json:"clusterDataCentre"`
+	ResourceGroup      string        `json:"resourceGroup"`
+	PeerVNet           string        `json:"peerVNet"`
+	PeerResourceGroup  string        `json:"peerResourceGroup"`
+	SubscriptionId     string        `json:"subscriptionId"`
+	VNet               string        `json:"vNet"`
+	PeerSubnet         string        `json:"peerSubnet"`
```

Comment:
Do we need `peerSubnet` here? I thought Azure peering was created after we moved to peering requests w/ multiple subnets?